test-data/unit/semanal-namedtuple.test

```diff
@@ -127,9 +127,11 @@ N = namedtuple('N', 1) # E: List literal expected as the second argument to name
 from collections import namedtuple
 N = namedtuple('N', ['x', 1]) # E: String literal expected as namedtuple() item
 
-[case testNamedTupleWithInvalidArgs]
+-- NOTE: The following code works at runtime but is not yet supported by mypy.
```

Comment:
Honestly, in Python 3.6 I'd love NamedTuple to take keyword args so you can say
```
Point = typing.NamedTuple('Point', x=int, y=int)
```
It requires Python 3.6+ because that promises to preserve keyword arg order.

But this would require support in typing too. (@ilevkivskyi?) Not for this PR obviously...